code/modules/discord/accountlink.dm

```diff
@@ -23,4 +23,30 @@
 		alert(usr, "Account link started. Please type the following in any channel '!verify yourckey' (Example: !verify nichlas0010)")
 	else // Account is already linked
 		alert("You already have the Discord Account [stored_id] linked to [usr.ckey]. If you need to have this reset, please contact an admin!","Already Linked","OK")
-		return
\ No newline at end of file
+		return
+
+// Verb to link forum accounts to BYOND accounts
+/client/verb/linkforum()
+	set category = "Admin"
+	set name = "Link Forum Account"
+	set desc = "Link your Forum account to your BYOND account."
+
+	if(!CONFIG_GET(string/webhook_key))
+		to_chat(src, "<span class='warning'>Error: Please contact your system administrator</span>")
+		return
+
+	var/datum/http_request/req = new()
+	req.prepare(RUSTG_HTTP_METHOD_POST, "https://forums.yogstation.net/api/linking/byond/[ckey]", "", list("XF-Api-Key" = CONFIG_GET(string/webhook_key)))
+	req.begin_async()
+	addtimer(CALLBACK(src, /client/proc/givelinkforum), 15 MINUTES, TIMER_STOPPABLE)
+	verbs -= /client/verb/linkforum
+
+	UNTIL(req.is_complete())
+	var/datum/http_response/response = req.into_response()
+	var/list/body = json_decode(response.body)
+	to_chat(src, "<a href='[body["url"]]'>Verify your account here!</a> Link will expire in 15 minutes.", confidential = TRUE)
+
+
+/client/proc/givelinkforum()
+	verbs += /client/verb/linkforum
+	to_chat(src, "<span class='notice'>Your forum verification link has expired, and the verb has been returned to you.</span>")
```

Comment:
You should probably add `You can ignore this if you have successfully linked your account`, since players might get confused if they link their account and then get this message.